packages/mds-utils/utils.ts

```diff
@@ -730,6 +730,13 @@ const testEnvSafeguard = () => {
   }
 }
 
+export const START_ONE_MONTH_AGO = now() - (now() % days(1)) - days(30)
+export const START_ONE_WEEK_AGO = now() - (now() % days(1)) - days(7)
+export const START_YESTERDAY = now() - (now() % days(1))
+export const START_NOW = now()
+export const START_TOMORROW = now() + (now() % days(1))
+export const START_ONE_MONTH_FROM_NOW = now() - (now() % days(1)) + days(30)
```

Comment:
MDS Utils is such a better place for this stuff to live, yay 🥳! I've got a couple other ideas that might make sense to do here too:
Do you think it'd make sense to strip the `START` prefix, and just call these constants e.g. `ONE_MONTH_AGO`? Also another idea (a tad excessive, but maybe kinda nice), we could annotate these with the `Timestamp` type (even though it's just an alias for `number`) to make it a little more expressive what they are when importing elsewhere.